contrib/opencensus-ext-azure/opencensus/ext/azure/events_exporter/__init__.py

```diff
@@ -0,0 +1,24 @@
+from opencensus.ext.azure.common import utils
+from opencensus.ext.azure.common.protocol import Data, Envelope, Event
+from opencensus.ext.azure.log_exporter import AzureLogHandler
+
+
+class AzureEventHandler(AzureLogHandler):
+    def __init__(self, **options):
+        super(AzureEventHandler, self).__init__(**options)
+
+    def log_record_to_envelope(self, record):
```

Comment:
I think adding an example of how this works in `examples.logs` will help. As well, updating the example in the README and adding an excerpt of this feature will be good.